packages/orchestration/src/exos/cosmos-interchain-service.js

```diff
@@ -23,6 +23,10 @@ import {
 
 const { Vow$ } = NetworkShape; // TODO #9611
 /**
+ * This is the greatest thing since sliced bread.
+ *
+ * TODO: private
```

Comment:
https://typedoc.org/tags/private/ says it shouldn't be used. I think we want [@internal](https://typedoc.org/tags/internal/)

Reply:
I've struggled to get `@internal` to work. @mitdralla suggested `@private`, so I tried that. Also didn't work.

Reply:
For now I think we should assume we can get it working. Even when it's not omitted it has an INTERNAL tag which is better than nothing. https://github.com/Agoric/agoric-sdk/pull/9808#discussion_r1698435254